user-interface/src/main/java/life/qbic/datamanager/views/general/CardLayout.java

```diff
@@ -0,0 +1,23 @@
+package life.qbic.datamanager.views.general;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.html.Div;
+
+/**
+ * Card Layout
+ * <p>
+ * Card styled Layout employed within all {@link Main} in the
+ * {@link life.qbic.datamanager.views.login.LoginLayout}
+ */
+public class CardLayout extends Div {
```

Comment:
What is a Card Layout? where is it used? How can it look. I am confused about this.

Reply:
I updated the JD for more clarity. In a nutshell we currently employ the same card-like style for all components during password reset, registration and login. To avoid having to copy paste the css over and over again, this DIV based component contains the style necessary to make it look like a hovering card containing the input fields. 

<img width="401" alt="Screenshot 2024-09-05 at 17 14 41" src="https://github.com/user-attachments/assets/78ee7c1c-3ef8-43b0-b14e-0895071ab9a6">
<img width="445" alt="Screenshot 2024-09-05 at 17 14 36" src="https://github.com/user-attachments/assets/88b0995d-3703-46ea-aa4b-113f0a224f39">

Reply:
> I updated the JD for more clarity. In a nutshell we currently employ the same card-like style for all components during password reset, registration and login. To avoid having to copy paste the css over and over again, this DIV based component contains the style necessary to make it look like a hovering card containing the input fields.

You do not need to copy paste anything but the css class name if I am not mistaken? I would vote in favor of composition over inheritance here. Components that extend card cannot extend anything else. If you want it to look like a card, why not only add the css class name. Shouldn't this be sufficient?

Reply:
Good point! For me it feels more like a feeling thing, we want to reuse the same styling for all the components in this context, so i'd like to have it as a general java class. But since it provides no functionality as of yet, we can also go for YAGNI and copy paste the css class. I adapted the PR accordingly and removed the cardlayout 🤔